airflow/www/views.py

```diff
@@ -798,11 +798,14 @@ def log(self):
         if PY2 and not isinstance(log, unicode):
             log = log.decode('utf-8')
 
+        if request.is_xhr:
```

Comment:
@zodiac `is_xhr` may not be the best way, but good enough for this small feature. We of course need it to evolve in to auto-refresh/auto-tailing thing, at that time we can invest on full fledged new api.